lib/api/controllers/collectionController.js

```diff
@@ -341,6 +319,31 @@ class CollectionController extends BaseController {
         return response;
       });
   }
+
+  /**
+   * @param {Request} request
+   * @returns {Promise.<Array>}
+   */
+  _createSpecificationList (index, collection, specifications) {
+    if (index && collection) {
+      const specification = formatSpecification(
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Move the declaration of "formatSpecification" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)